windows/hid.c

```diff
@@ -971,10 +1037,11 @@ DWORD WINAPI hid_internal_notify_callback(HCMNOTIFICATION notify, PVOID context,
 		}
 
 		/* Get and remove this device from the device list */
-		for (struct hid_device_info** current = &hid_hotplug_context.devs; *current; current = &(*current)->next) {
+		for (struct hid_device_info **current = &hid_hotplug_context.devs; *current;
```

Comment:
no need of newline here

Reply:
looks like some ("default"?) IDE settings